src/app/sw.ts

```diff
@@ -0,0 +1,59 @@
+import { defaultCache } from '@serwist/next/worker'
```

Comment:
Q: i have no idea what thiis file does - can  we add a docstring explaining it?

Reply:
Yes it's actually boilerplate code from serwist. DefaultCache is just a set or recommended cache strategies for the pwa. Adding link to docs ✅